web/app/configs/lang.json

```diff
@@ -387,18 +387,24 @@
         }
     },
     "it": {
-        "wind": "eolico",
-        "solar": "fotovoltaico",
-        "hydro": "idroelettrico",
-        "hydro storage": "immag. idro.",
-        "biomass": "biomassa",
-        "nuclear": "nucleare",
-        "geothermal": "geotermale",
-        "gas": "gas",
-        "coal": "carbone",
-        "oil": "petrolio",
-        "unknown": "sconosciuto",
-
+        "wind": "Eolico",
+        "solar": "Fotovoltaico",
+        "hydro": "Idroelettrico",
+        "hydro storage": "Accumulo idro.",
+        "biomass": "Biomassa",
+        "nuclear": "Nucleare",
+        "geothermal": "Geotermale",
+        "gas": "Gas",
+        "coal": "Carbone",
+        "oil": "Petrolio",
+        "unknown": "Sconosciuto",
```

Comment:
You should remove a couple of those entries (that I removed during the last changes).